addons/l10n_tr/data/account_tax_template_data.xml

```diff
@@ -35,16 +35,16 @@
         ]"/>
     </record>
```

Comment:
This is what I would suggest,
create the new tax:

```xml
    <record id="tr_kdv_satis_sale_20" model="account.tax.template">
        <field name="sequence">12</field>
        <field name="description">KDV %20(sale)</field>
        <field name="name">KDV %20(sale)</field>
        <field name="price_include" eval="0"/>
        <field name="amount">20</field>
        <field name="amount_type">percent</field>
        <field name="type_tax_use">sale</field>
        <field name="chart_template_id" ref="l10ntr_tek_duzen_hesap"/>
        <field name="tax_group_id" ref="tax_group_kdv_20"/>
        <field name="invoice_repartition_line_ids" eval="[(5,0,0),
            (0,0, {
                'factor_percent': 100,
                'repartition_type': 'base',
            }),
            (0,0, {
                'factor_percent': 100,
                'repartition_type': 'tax',
                'account_id': ref('tr391'),
            }),
        ]"/>
        <field name="refund_repartition_line_ids" eval="[(5,0,0),
            (0,0, {
                'factor_percent': 100,
                'repartition_type': 'base',
            }),
            (0,0, {
                'factor_percent': 100,
                'repartition_type': 'tax',
                'account_id': ref('tr191'),
            }),
        ]"/>
    </record>
```

don't delete the old one (`tr_kdv_satis_purchase_18`), but instead add the field
`<field name="active" eval="False"/> to it